db/seeds.rb

```diff
@@ -43,7 +44,9 @@
   reason_to_donate: 'I want to help a girl receive an excellent education',
   slug: 'annie-wright-schools',
   info_url: 'http://www.aw.org',
-  donation_url: 'https://aw.myschoolapp.com/Page/Support/Make-a-Gift?siteId=693&ssl=1'
+  donation_url: 'https://aw.myschoolapp.com/Page/Support/Make-a-Gift?siteId=693&ssl=1',
+  sort_order: 0,
+  featured_content: 'This is very important to us and here is why'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.